src/test/java/Config.java

```diff
@@ -0,0 +1,15 @@
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.TextComponent;
+import net.kyori.adventure.text.format.NamedTextColor;
+import space.arim.dazzleconf.annote.ConfDefault;
+import space.arim.dazzleconf.annote.ConfKey;
+
+interface Config {
+    static TextComponent yes() {
+        return Component.text("HELLO", NamedTextColor.GOLD).append(Component.text("BYE", NamedTextColor.RED));
+    }
+
+    @ConfKey("component")
+    @ConfDefault.DefaultObject("yes")
```

Comment:
By the way, you can use `@DefaultString("&6HELLO &cBYTE")` and dazzleconf will parse the default string just like a user-specified string

Reply:
Since  you're not replying on discord, I need you to wipe and recreate the database, so we can launch the fork and finish  with Clans @A248